doc/source/cluster/configure-manage-dashboard.md

```diff
@@ -0,0 +1,304 @@
+:::
+
+:::{tab-item} Kuberay
+
+The KubeRay operator makes the dashboard available via a Service targeting the Ray head pod, named ``<RayCluster name>-head-svc``. You can access the
+dashboard from within the Kubernetes cluster at ``http://<RayCluster name>-head-svc:8265``.
+
+**Add Ingress rule**
+
+**Port forwarding** <br/>
+You can also view the dashboard from outside the Kubernetes cluster by using port-forwarding:
+
+```{admonition} Note
+:class: note
+It's not recommended to use port forwarding if you need have specific network security requirements. Follow the instructions below to add ingress rules to expose Ray Dashboard
+```
+
+```shell
+$ kubectl port-forward service/raycluster-autoscaler-head-svc 8265:8265
+```
+For more information about configuring network access to a Ray cluster on Kubernetes, see the {ref}`networking notes <kuberay-networking>`.
+
+:::
+
+::::
+
+
+## Running Behind a Reverse Proxy
+
+Ray dashboard should work out-of-the-box when accessed via a reverse proxy. API requests don't need to be proxied individually.
+
+Always access the dashboard with a trailing ``/`` at the end of the URL.
+For example, if your proxy is set up to handle requests to ``/ray/dashboard``, view the dashboard at ``www.my-website.com/ray/dashboard/``.
+
+The dashboard now sends HTTP requests with relative URL paths. Browsers will handle these requests as expected when the ``window.location.href`` ends in a trailing ``/``.
+
+This is a peculiarity of how many browsers handle requests with relative URLs, despite what [MDN](https://developer.mozilla.org/en-US/docs/Learn/Common_questions/What_is_a_URL#examples_of_relative_urls) defines as the expected behavior.
+
+Make your dashboard visible without a trailing ``/`` by including a rule in your reverse proxy that redirects the user's browser to ``/``, i.e. ``/ray/dashboard`` --> ``/ray/dashboard/``.
+
+Below is an example with a [traefik](https://doc.traefik.io/traefik/getting-started/quick-start/) TOML file that accomplishes this:
+
+```yaml
+[http]
+  [http.routers]
+    [http.routers.to-dashboard]
+      rule = "PathPrefix(`/ray/dashboard`)"
+      middlewares = ["test-redirectregex", "strip"]
+      service = "dashboard"
+  [http.middlewares]
+    [http.middlewares.test-redirectregex.redirectRegex]
+      regex = "^(.*)/ray/dashboard$"
+      replacement = "${1}/ray/dashboard/"
+    [http.middlewares.strip.stripPrefix]
+      prefixes = ["/ray/dashboard"]
+  [http.services]
+    [http.services.dashboard.loadBalancer]
+      [[http.services.dashboard.loadBalancer.servers]]
+        url = "http://localhost:8265"
+```
+
+## Disabling the Dashboard
+
+Dashboard is included if you use `ray[default]`, `ray[air]` or {ref}`other installation commands <installation>` and automatically started.
+
+To disable the dashboard, use the following arguments `--include-dashboard`.
+
+::::{tab-set}
+
+:::{tab-item} Single-node local cluster
+
+**Start the cluster explicitly with CLI** <br/>
```

Comment:
```suggestion
**Start the cluster explicitly with the CLI** <br/>
```

Reply:
Are you open to shortening the tab titles in the tab to something like `CLI` and an equivalent term for  `ray.init`? I don't know what the appropriate term for the `ray.init` parameter would be, but it would make the tab titles cleaner.

Reply:
Which tab title are you referring to?  `Start the cluster explicitly with CLI` is not a tab title.

Reply:
I see. I was confused. Please ignore.